data-raw/process_data_set_danica.R

```diff
@@ -0,0 +1,24 @@
+library(tidyverse)
+library(stringr)
+library(lubridate)
+library(janitor)
+library(usethis)
+
+historical_RAPTOR_by_player <- read_csv("data-raw/nba-raptor/historical_RAPTOR_by_player.csv") 
+
+modern_RAPTOR_by_player <- read_csv("data-raw/nba-raptor/modern_RAPTOR_by_player.csv") 
+
+historical_RAPTOR_by_team <-read_csv("data-raw/nba-raptor/historical_RAPTOR_by_team.csv") %>%
+  mutate(
+    season_type = as.factor(season_type)
+    )
+
+modern_RAPTOR_by_team <-read_csv("data-raw/nba-raptor/modern_RAPTOR_by_team.csv") %>%
+  mutate(
+    season_type = as.factor(season_type)
+  )
```

Comment:
Do the same here: create a single data frame `raptor_by_team` that is a `bind_rows()` of `historical_raptor_by_team` and `modern_raptor_by_team`, that also has a new variable `era` indicating which era each row is from.